model/workflow.go

```diff
@@ -58,7 +58,7 @@ type BaseWorkflow struct {
 	// Key Domain-specific workflow identifier
 	Key string `json:"key,omitempty" validate:"omitempty,min=1"`
 	// Workflow name
-	Name string `json:"name" validate:"required"`
+	Name string `json:"name" validate:"omitempty"`
```

Comment:
If the validate rule is omitempty, should we just remove this tag？

Reply:
better keep just `json:"name,omitempty"`